modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/ScheduleModal.js

```diff
@@ -69,7 +69,10 @@ export default function ScheduleModal({
 				</p>
 
 				<div className={classnames({'has-error': invalidDate})}>
+					<label>{Liferay.Language.get('date-and-time')}</label>
+
 					<ClayDatePicker
+						firstDayOfWeek={1}
```

Comment:
Should the first day of the week always be the same? Isn't it locale dependent?

Reply:
Do you know if there is a method/function that returns this depending on the locale?

Reply:
See example with `Calendar` in `input_scheduler.jsp`

Reply:
I removed this for now, and will check it in https://liferay.atlassian.net/browse/LPS-196180